packages/p2p/app/index.js

```diff
@@ -0,0 +1,68 @@
+/* P2P peer discovery server */
+
+const WSServer = require('rpc-websockets').Server
+
+function conf (name, defaultValue) {
+  var value = process.env[name]
+  if (value === undefined) {
+    value = defaultValue
+  }
+  if (value === undefined) {
+    throw new Error(`Required configuration "${name}" not found.`)
+  }
+  return value
+}
+
+const host = conf('PEER_DISCOVERY_HOST', 'localhost')
+const port = conf('PEER_DISCOVERY_PORT', '8889')
+
+const server = new WSServer({ host, port })
+
+console.log(`Peer discovery server listening on ws://${host}:${port}`)
+
+const ROOMS = {}
```

Comment:
I would probably down case this. I generally reserve all caps for global constants that feel like configuration.